plugin/sampling/strategystore/static/strategy_store_test.go

```diff
@@ -51,22 +72,17 @@ func mockStrategyServer() *httptest.Server {
 			return
 
 		default:
-			data, err := ioutil.ReadFile("fixtures/strategies.json")
-			if err != nil {
-				w.WriteHeader(500)
-				return
-			}
 			w.WriteHeader(200)
 			w.Header().Set("Content-Type", "application/json")
-			w.Write(data)
+			w.Write(strategiesJSON)
 		}
 	}
 	return httptest.NewServer(http.HandlerFunc(f))
 }
 
-func TestStrategyStore(t *testing.T) {
+func TestStrategyStoreWithFile(t *testing.T) {
 	_, err := NewStrategyStore(Options{StrategiesFile: "fileNotFound.json"}, zap.NewNop())
-	assert.EqualError(t, err, "failed to read strategies file fileNotFound.json: open fileNotFound.json: no such file or directory")
+	assert.Contains(t, err.Error(), "failed to read strategies file fileNotFound.json: open fileNotFound.json: no such file or directory")
```

Comment:
```suggestion
	assert.Contains(t, err.Error(), "failed to read strategies file fileNotFound.json")
```
only include the string that we generate, not the stdlib portion